.github/scripts/Tests/frame-credit-ledger-orchestrator.Tests.ps1

```
@@ -625,6 +625,219 @@ integrity_checks:
+                        throw 'simulated body-fetch failure'
+                    }
+                    $global:LASTEXITCODE = 0
+                    # Parse the PrBodyJson to get the body string, then wrap with empty comments.
+                    try {
+                        $parsedBody = $resolvedPrBodyJson | ConvertFrom-Json
+                        if ($null -ne $parsedBody -and $null -ne $parsedBody.body) {
+                            # Already has body+comments shape.
+                            return $resolvedPrBodyJson
+                        }
+                    }
+                    catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)